py/server/deephaven/parquet.py

```diff
@@ -33,27 +71,37 @@ class ColumnInstruction:
 
 
 def _build_parquet_instructions(
-    col_instructions: List[ColumnInstruction] = None,
-    compression_codec_name: str = None,
-    max_dictionary_keys: int = None,
-    max_dictionary_size: int = None,
-    is_legacy_parquet: bool = False,
-    target_page_size: int = None,
-    is_refreshing: bool = False,
-    for_read: bool = True,
-    force_build: bool = False,
+        col_instructions: List[ColumnInstruction] = None,
```

Comment:
All of the None should be `Optional` - although, I see it was "incorrect" before. Ask @jmao-denver if we want to be stricter about this.